### PR TITLE
Fix reusable forms checkbox radio button issue on front-end

### PR DESCRIPTION
**Pull request by supriyajjadhav**
[Trello](https://trello.com/c/znk20jJa/224-package-testing-1220)

### Description
- Fixed Radio Button, Toggle, Accept, and Checkbox block issues on the front-end when we add these forms. 

#### Test Cases
- Add form on the editor 
- Add radio button, checkbox, toggle, and accept field.
- Add that form to reusable block
- Add a new form on the editor
- Add radio button, checkbox, toggle, and accept field.
- check the front-end all the fields and stylings are working properly.

### Types of changes
- Added reusable block id and added CSS to forms block.

### How has this been tested?
Checked whether it is working on front-end or not.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] I've included any necessary tests

### CODE REVIEW

**Comment by RajkiranBagal** [comment]
@supriyajjadhav Also check the normal case, without reusable block.

**Comment by supriyajjadhav** [comment]
> @supriyajjadhav Also check the normal case, without reusable block.

Checked... Working Fine

**Comment by RajkiranBagal** [comment]
> > @supriyajjadhav Also check the normal case, without reusable block.
> 
> Checked... Working Fine

Not at my end - https://share.getcloudapp.com/6qu8KWpv

File: classes/class-uagb-block-helper.php [hunk 1]
```diff
@@ -5284,6 +5284,139 @@ public static function get_forms_css( $attr, $id ) {
...
+								$selectors[' .uagb-forms-main-form .uagb-slider:before'] = array(
+									'width'  => UAGB_Helper::get_css_value( '20' + $attr['toggleWidthSize'] - $attr['inputborderWidth'] / 2, 'px' ),
+									'height' => UAGB_Helper::get_css_value( '20' + $attr['toggleHeightSize'] - $attr['inputborderWidth'], 'px' ),
+								);
+								$selectors[' .uagb-switch input:checked + .uagb-slider'] = array(
+									'background-color' => $attr['toggleActiveColor'],
+								);
+								$selectors[' .uagb-switch input:focus + .uagb-slider']   = array(
+									'box-shadow' => '0 0 1px' . $attr['toggleActiveColor'],
+								);
```
Comment (RajkiranBagal) [inline classes/class-uagb-block-helper.php:5374]: Please eliminate the duplicate code.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: classes/class-uagb-block-helper.php (hunk 1 above)
Comment (supriyajjadhav) [reply classes/class-uagb-block-helper.php]: Done

File: classes/class-uagb-block-helper.php [hunk 2]
```diff
@@ -5284,122 +5284,137 @@ public static function get_forms_css( $attr, $id ) {
...
+				if ( isset( $block['blockName'] ) && 'core/block' === $block['blockName'] ) {
+					$block_id = ( isset( $block['attrs']['ref'] ) ) ? $block['attrs']['ref'] : 0;
+					if ( $block_id ) {
+						$content = get_post_field( 'post_content', $block_id );
 
-					if ( 'boxed' === $attr['formStyle'] ) {
-						$selectors[' .uagb-forms-main-form  .uagb-forms-checkbox-wrap input[type=checkbox] + label:before'] = array(
-							'border'        => UAGB_Helper::get_css_value( $attr['inputborderWidth'], 'px' ) . ' ' . $attr['inputborderStyle'] . ' ' . $attr['inputborderColor'],
-							'border-radius' => UAGB_Helper::get_css_value( $attr['inputborderRadius'], 'px' ),
+						$reusable_blocks      = version_compare( $wp_version, '5', '>=' ) ? parse_blocks( $content ) : gutenberg_parse_blocks( $content );
```
Comment (RajkiranBagal) [inline classes/class-uagb-block-helper.php:5294]: This is also declared above.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: classes/class-uagb-block-helper.php (hunk 2 above)
Comment (supriyajjadhav) [reply classes/class-uagb-block-helper.php:5294]: Right! In reusable block case, we dont find our blocks list inside the first $reusable_block. So inside the first for we are checking if the block id is present then we have added $content = get_post_field( 'post_content', $block_id ); and passing this in $reusable_block variable to get blocks data.

Comment (RajkiranBagal) [reply classes/class-uagb-block-helper.php:5294]: Yes, Right! Totally forgot.

**Review by RajkiranBagal — APPROVED** [review]

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: classes/class-uagb-block-helper.php [hunk 3]
```diff
@@ -5235,11 +5235,11 @@ public static function get_forms_css( $attr, $id ) {
 			$selectors   = array();
 			$m_selectors = array();
 			$t_selectors = array();
-			$content     = get_post_field( 'post_content' );
+
+			$content = get_post_field( 'post_content' );
```
[the remark this answers is not in the document]
Comment (supriyajjadhav) [reply classes/class-uagb-block-helper.php:5239]: In the reusable block case, 
- we are not able to get a blocklist using $content = get_post_field( 'post_content' ); we get this --> https://share.getcloudapp.com/WnuYedgQ
- When we add this $content in $reusable_blocks = version_compare( $wp_version, '5', '>=' ) ? parse_blocks( $content ) : gutenberg_parse_blocks( $content ); we get --> https://share.getcloudapp.com/d5u1BYZm

[the remark this answers is not in the document]
Comment (vrundakansara) [reply classes/class-uagb-block-helper.php:5239]: Let's remove it @supriyajjadhav

**Review by vrundakansara — CHANGES_REQUESTED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**